prow/images/kyma-integration/Dockerfile

```diff
@@ -148,4 +147,4 @@ RUN wget -q https://dl.google.com/dl/cloudsdk/channels/rapid/downloads/google-cl
     --usage-reporting=false && \
     gcloud components install alpha beta kubectl docker-credential-gcr && \
     gcloud info | tee /workspace/gcloud-info.txt && \
-    mv /google-cloud-sdk/bin/kubectl.${CLUSTER_VERSION} /google-cloud-sdk/bin/kubectl
+    mv /google-cloud-sdk/bin/kubectl.${K8S_VERSION} /google-cloud-sdk/bin/kubectl
```

Comment:
You need this to be in sync with cluster version to include all changes from new version. If you use K8S_VERSION variable here and do not override it with build args, you will get kubectl 1.16.